resources/idlharness.js

```diff
@@ -2855,17 +2856,27 @@ IdlInterface.prototype.test_interface_of = function(desc, obj, exception, expect
         return;
     }
 
+    var unexposed_properties = [];
     for (var i = 0; i < this.members.length; i++)
     {
         var member = this.members[i];
         if (member.untested) {
             continue;
         }
-        if (!exposed_in(exposure_set(member, this.exposureSet))) {
-            subsetTestByKey(this.name, test, function() {
-                assert_equals(exception, null, "Unexpected exception when evaluating object");
-                assert_false(member.name in obj);
-            }.bind(this), this.name + " interface: " + desc + ' must not have property "' + member.name + '"');
+        if (!exposed_in(exposure_set(member, this.exposureSet)))
+        {
+            if (unexposed_properties.includes(member.name))
+            {
+                assert_true(true, `Already created test for "${member.name}"`);
```

Comment:
What's the point of this? I don't think we're even in a `test` here

Reply:
Thanks for your reviewing. 
I updated PR by removing it.